docs/docs/Components/components-vector-stores.md

```diff
@@ -66,6 +66,40 @@ For more information, see the [DataStax documentation](https://docs.datastax.com
 | vector_store | Vector Store | Built Astra DB vector store |
 | search_results | Search Results | Results of the similarity search as a list of Data objects |
 
+
+## AstraDB Graph vector store
+
+This component implements a Vector Store using AstraDB with graph capabilities.
+For more information, see the [DataStax documentation](https://docs.datastax.com/en/astra-db-serverless/tutorials/graph-rag.html).
+
+### Inputs
+
+| Name | Display Name | Info |
+|------|--------------|------|
+| collection_name | Collection Name | The name of the collection within AstraDB where the vectors will be stored (required) |
+| token | Astra DB Application Token | Authentication token for accessing AstraDB (required) |
+| api_endpoint | API Endpoint | API endpoint URL for the AstraDB service (required) |
+| search_input | Search Input | Query string for similarity search |
+| ingest_data | Ingest Data | Data to be ingested into the vector store |
+| namespace | Namespace | Optional namespace within AstraDB to use for the collection |
+| embedding | Embedding Model | Embedding model to use |
+| metric | Metric | Distance metric for vector comparisons (options: "cosine", "euclidean", "dot_product") |
+| setup_mode | Setup Mode | Configuration mode for setting up the vector store (options: "Sync", "Async", "Off") |
+| pre_delete_collection | Pre Delete Collection | Boolean flag to determine whether to delete the collection before creating a new one |
+| number_of_results | Number of Results | Number of results to return in similarity search (default: 4) |
+| search_type | Search Type | Search type to use (options: "Similarity", "Graph Traversal", "Hybrid") |
+| traversal_depth | Traversal Depth | Maximum depth for graph traversal searches (default: 1) |
+| search_score_threshold | Search Score Threshold | Minimum similarity score threshold for search results |
+| search_filter | Search Metadata Filter | Optional dictionary of filters to apply to the search query |
+
+### Outputs
+
+| Name | Display Name | Info |
+|------|--------------|------|
+| vector_store | Vector Store | Built AstraDB Graph vector store |
```

Comment:
I don't understand the third column. Is it the name of an AstraDB graph vector store that has been built? It's not clear.